docs/whats-new/staging-release-notes.md

```diff
@@ -2,6 +2,50 @@
 
 > **Tip**: Click the triangle to view more details about the change.
 
+## Staging release 12 April 2023
+
+Frontend version: 1.0.0-20230406.0949 | Backend version: 1.58.2-230406.0515
+
+**Feature** 
+
+<details>
+<summary style="font-size:20px;font-weight:bold">Search bar added on TechPass portal</summary>
+
+We have updated TechPass portal with a new search bar feature that allows users to search for specific applications. Additionally, tenants can also use the search bar to search for applications and OTP Credentials.
+
+</details>
+
+**Changes** 
+
+<details>
+<summary style="font-size:20px;font-weight:bold"> Updated tenants' restriction on TechBiz application role assignment & groups management in TechPass </summary>
+
+Previously, tenants were not allowed to manage their applications' role assignments on TechPass if they had onboarded to TechBiz. However, we understand that some applications may not be on TechBiz, so we have updated the restriction to be applied at the application-level instead. 
```

Comment:
It took me some time to understand this:
**Previously, tenants were not allowed to manage their applications' role assignments on TechPass if they had onboarded to TechBiz. However, we understand that some applications may not be on TechBiz**

Perhaps we explain more clearly the differencedifference between tenant-wide level vs application level TechBiz onboarding.

Reply:
@yqyeoh could you quickly provide me the difference

Reply:
by identifying TechBiz onboarding at the tenant level, we are restricting tenants from managing the app role assignments for ALL of their applications on TechPass (as we are expecting they should all be managed on TechBiz); by moving down to application level, we are imposing the restriction per application (i.e. on TechPass, tenants cannot manage the app role assignments for applications that are on TechBiz; whereas they will still be able to manage the app role assignments for applications that are not on TechBiz)

Reply:
updated in new commit